docker/runtime/arm/rund.arm32v7.sh

```diff
@@ -3,7 +3,7 @@
 echo '=> detecting IP'
 export IP=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
 export IOT_DEVICE_HOSTNAME="$IP"
-export AGENT_IMAGE="mcr.microsoft.com/azureiotedge-agent:1.0@sha256:3fdb80a6dfe1fbcbf11cfcea56dda79d1d779490c66d394a2b3f388d74ab0c26"
+export AGENT_IMAGE="mcr.microsoft.com/azureiotedge-agent:1.1@sha256:84d81e0527799e903fca9602c56efcbc43ce6f90eaeb7316fddc7d663d40d1b1"
```

Comment:
>1.1 [](start = 57, length = 3)

I think we should revert the runtime images version to 1.0 in the docker files.

Reply:
reverted back in commit https://github.com/Azure/iotedgedev/pull/466/commits/4f2deafc933ae18895d3c17e1dacc8b47c89b5a3